xbmc/GUIUserMessages.h

```diff
@@ -103,6 +103,10 @@
 // Message sent to tell the GUI to update a single item
 #define GUI_MSG_UPDATE_ITEM           GUI_MSG_USER + 29
 
+  // Flags for GUI_MSG_UPDATE_ITEM message
```

Comment:
great! can you extract this bit (and related media window change) and submit it separately please. to keep the pr on topic.

Reply:
Separate PR https://github.com/xbmc/xbmc/pull/14998 for this.